### PR TITLE
micaforeveryone: Add version 0.0.4.1

### PR DESCRIPTION
**Pull request by sitiom**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/micaforeveryone.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.0.4.1",
+    "description": "Enables the backdrop effect on titlebar of win32 apps on Windows 11",
+    "homepage": "https://github.com/minusium/MicaForEveryone",
+    "license": "MIT",
+    "architecture": {
```
Comment (issaclin32) [inline bucket/micaforeveryone.json:6]:
```suggestion
    "pre_install": "if ([Environment]::OSVersion.Version.Build -lt 22000) { error 'Windows 11 is required'; break }",
    "architecture": {
```

Comment (sitiom) [reply bucket/micaforeveryone.json:6]: Interesting quirk.

**Comment by rashil2000** [comment]
@sitiom a new version for this was released with a lot of breaking changes. Can you update it in a PR?

**Pull request merged**